cfgov/v1/models/molecules.py

```diff
@@ -31,6 +33,27 @@ class ImageText5050(blocks.StructBlock):
     link_url = blocks.URLBlock(required=False)
     link_text = blocks.CharBlock(max_length=100, required=False)
 
+    def clean(self, data):
+        ordered_tuples = data.items()
```

Comment:
So, because **clean()** is getting overridden, you need to add `cleaned_data = super(ContactForm, self).clean()` unless you plan on doing the work that **clean()** also does. This comes with the added benefit of having the dictionary `cleaned_data` that has python objects to use.